metashare/api/views.py

```diff
@@ -102,3 +102,6 @@ class ScratchOrgViewSet(viewsets.ModelViewSet):
     queryset = ScratchOrg.objects.all()
     filter_backends = (DjangoFilterBackend,)
     filterset_class = ScratchOrgFilter
+
+    def perform_destroy(self, instance):
+        instance.queue_delete()
```

Comment:
When we start a deletion process, we should store that data somewhere on the model so that after a page-refresh the front end still knows that it's being deleted. The most obvious to me would be setting a `deletion_queued_at` timestamp on the model and saving it.